Makefile

```diff
@@ -1,6 +1,7 @@
 ONYX_DIR=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 USER_ID=$(shell id -u)
 GROUP_ID=$(shell id -g)
+COMPOSER_ARGS=--ignore-platform-reqs
```

Comment:
That's already the case : https://github.com/Onyx-org/skeleton/blob/php71/composer.json#L8